CRP.Mvc/Views/Item/Details.cshtml

```diff
@@ -0,0 +1,135 @@
+﻿@using CRP.Controllers.Helpers
+@using CRP.Core.Helpers
+@using Microsoft.Azure
+@model CRP.Controllers.ViewModels.ItemDetailViewModel
+
+@{
+    ViewBag.Title = "UC Davis Registration";
+    Layout = "~/Views/Shared/_PublicLayout.cshtml";    
+}
+
+
+
+<div class="main">
+    <div class="eventbanner">
+        <div class="eventbanner-image">
+            <img src="@Url.Action("GetImage", "Item", new {id = Model.Item.Id})" alt="x" />

```

Comment:
@jSylvestre if you are going to put in an alt at least make it something like "Event Banner"